### ISSUE TITLE
network error while verifying the vc

### ISSUE THREAD

**Issue by Muthurajj** [docknetwork/vcdm-verifier#35]
Hi , I have generated VC with unireslover while verifying i am getting "A DID document lookup was successful, but the DID in question does not exist. This is different from a network error" this error do you have any idea ??

This is my diddocument:
{
  "state": "finished",
  "secret": null,
  "didDocument": {
    "@context": "https://www.w3.org/ns/did/v1",
    "authentication": [],
    "service": [
      {
        "type": "ServiceEndpointProxyService",
        "serviceEndpoint": "https://sysveda.co.in"
      }
    ],
    "verificationMethod": [
      {
        "type": "Ed25519VerificationKey2018",
        "publicKeyBase64": "de8777a28f8da1a74e7a13090ed974d879bf692d001cddee16e4cc9f84b60580"
      }
    ],
    "id": "did:web:did-web.uniregistrar.io:dadaa7f5-47f9-430f-a4a7-1074bd1c6508"
  },
  "did": "did:web:did-web.uniregistrar.io:dadaa7f5-47f9-430f-a4a7-1074bd1c6508"
}

This is the generated Vc:

{
    "@context": [
        "https://www.w3.org/2018/credentials/v1",
        "https://www.w3.org/2018/credentials/examples/v1"
    ],
    "type": [
        "UniversityDegreeCredential",
        "VerifiableCredential"
    ],
    "issuer": "did:web:did-web.uniregistrar.io:dadaa7f5-47f9-430f-a4a7-1074bd1c6508",
    "issuanceDate": "2019-06-16T18:56:59Z",
    "expirationDate": "2023-06-17T18:56:59Z",
    "credentialSubject": {
        "degree": {
            "name": "Bachelor of Science and Arts",
            "type": "BachelorDegree"
        }
    },
    "proof": {
        "type": "Ed25519Signature2018",
        "created": "2022-09-26T05:25:44Z",
        "proofPurpose": "assertionMethod",
        "verificationMethod": "did:web:did-web.uniregistrar.io:dadaa7f5-47f9-430f-a4a7-1074bd1c6508#keys-1",
        "jws": "eyJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdLCJhbGciOiJFZERTQSJ9..pASJSRwohIhieNhMjghlx9Wc_WoAEqZk-Gq6E3YyghIrS-tzq3lL44XA30CVDHtwaolNW9EcQIEUVpq1eo6FCg"
    }
}

**Comment by SamHellawell** [comment]
Yep, there was an update to the universal resolver that wasnt caught. The DID is now resolvable, however the verifier/SDK still doesnt support `publicKeyBase64` yet. I will look into this.

**Comment by SamHellawell** [comment]
Can you share how you created the credential? Locally I am able to make some changes to get your public key and DID read - however it says invalid signature. Secondly, I think there are missing JSON-LD contexts which causes key lookup to fail. There is no key in the DID with the id of "xyz#keys-1" which is causing the lookup of the verification method to fail.

**Comment by Muthurajj** [comment]
I have created my VC using https://github.com/danubetech/verifiable-credentials-java library and i just add DID from uniresolver.

**Comment by SamHellawell** [comment]
Please be specific, ideally give code so I can try to reproduce. Either way, the initial SDK issue has been resolved and the verifier can now read your DID - and even use publicKeyBase64. Problem is, your JSON-LD document is missing contexts to allow that to happen. It's something you will have to research/create to support the publicKeyBase64 property. Other problem is that DID doesnt define any IDs for the keys in it, so its not usable for JSON-LD verification as a W3C credential.

In short:
- Create a DID another way where the document is correct (see example of dock did resolution or this did:web - https://github.com/docknetwork/sdk/blob/master/example/didweb.js#L39
- Create a custom JSON-LD context to support publicKeyBase64 (if your DID document still has that, not all DID providers do. Yours is the first I've seen)

**Comment by Muthurajj** [comment]
@SamHellawell thanks it really useful
